apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/service/AppService.java

```diff
@@ -139,4 +166,46 @@ public EnvClusterInfo createEnvNavNode(Env env, String appId) {
     return node;
   }
 
+  @Transactional
+  public void deleteByAppId(String appId) {
```

Comment:
I think besides deleting the app from apollo portal db, we also need to delete it from apollo config db in each environments, this involves calling some service in each environments' admin service, and we need to keep an eye how to achieved the consistency in distributed environments.